test/hs/Test/Ganeti/HTools/Node.hs

```diff
@@ -134,6 +134,7 @@ genEmptyOnlineNode =
                        , Node.rMemForth = 0
                        , Node.pRem = 0
                        , Node.pRemForth = 0
+                       , Node.xMem = 0
```

Comment:
But isn't xMem the difference between the free memory according to the record and the actual free memory? If we update the state of record, the function xMem would change without needing any update, right?

Anyway, it would probably be a better idea to change that in another patch. So let's keep it as is for now.